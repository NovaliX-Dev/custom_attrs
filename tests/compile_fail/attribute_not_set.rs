use custom_attrs::CustomAttrs;

#[derive(CustomAttrs)]
#[attr(pub a: usize)]
enum Enum {
    Variant1,

    #[attr(a = 3)]
    Variant2
}

fn main() {
    // let _a = Enum::Variant1.get_a();
}