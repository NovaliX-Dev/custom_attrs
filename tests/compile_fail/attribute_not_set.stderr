error: Value not set for `a`.
 --> tests/compile_fail/attribute_not_set.rs:6:5
  |
6 |     Variant1,
  |     ^^^^^^^^
